rs/sns/governance/src/governance/advance_target_sns_version_tests.rs

```diff
@@ -351,3 +635,177 @@ fn test_perform_advance_target_version() {
         assert_eq!(result, expected_result, "{}", label);
     }
 }
+
+fn add_calls_for_initiate_upgrade(
```

Comment:
set_environment_call_expectations_and_responses_for_initiate_upgrade

Reply:
changed to somethign else!